### ISSUE TITLE
various no-unknown-arguments-for-builtin-components errors

### ISSUE THREAD

**Issue by sebastianhelbig** [ember-template-lint/ember-template-lint#1820]
1) All this arguments are working as expected. Imho it should not show any error. Or did I miss something?
2) @invokeAction is added to LinkTo by https://github.com/Kuzirashi/ember-link-action. What should be done to support this?

**Input**
```
error  "@autocomplete" is unknown argument for <Input /> component. Did you mean "@value"?  no-unknown-arguments-for-builtin-components
error  "@class" is unknown argument for <Input /> component. Did you mean "@escape-press"?  no-unknown-arguments-for-builtin-components
error  "@required" is unknown argument for <Input /> component. Did you mean "@escape-press"?  no-unknown-arguments-for-builtin-components
error  "@spellcheck" is unknown argument for <Input /> component. Did you mean "@checked"?  no-unknown-arguments-for-builtin-components
```

**LinkTo**
```
error  "@class" is unknown argument for <LinkTo /> component. Did you mean "@activeClass"?  no-unknown-arguments-for-builtin-components
error  "@invokeAction" is unknown argument for <LinkTo /> component. Did you mean "@activeClass"?  no-unknown-arguments-for-builtin-components
```

**Comment by rwjblue** [comment]
> All this arguments are working as expected. Imho it should not show any error. Or did I miss something?

The rule is specifically using the arguments listing detailed in [emberjs/rfcs#707](https://github.com/emberjs/rfcs/blob/master/text/0707-modernize-built-in-components-2.md#summary). All of the other arguments will be issuing deprecation messages in Ember 3.27 and higher.

> @invokeAction is added to LinkTo by https://github.com/Kuzirashi/ember-link-action. What should be done to support this?

Due to the changes proposed in [emberjs/rfcs#671](https://github.com/emberjs/rfcs/blob/master/text/0671-modernize-built-in-components-1.md) I think that https://github.com/Kuzirashi/ember-link-action should be updated to avoid mutating the built-in `LinkTo`.

---

Ultimately, I don't think this is a bug. The rule seems to be working as intended (and is helping you identify issues that you will want to address in your codebase as you progress through newer Ember versions).

**Comment by rwjblue** [comment]
FWIW, I've also created https://github.com/ember-template-lint/ember-template-lint/issues/1821 to track making these error messages better.

**Comment by sebastianhelbig** [comment]
Thanks a lot @rwjblue for the detailed explanation.

**Comment by rwjblue** [comment]
No problem! You might want to take a look at the new `--update-todo` feature to let you mark these failures as "todos" (and buy some time to fix them). [There are some docs here](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/todos.md).
